Assets/Scripts/Movement/Move.cs

```diff
@@ -0,0 +1,14 @@
+﻿using UnityEngine;
+
+namespace Movement
+{
+    public abstract class Move : MonoBehaviour
+    {
+        private void Update()
+        {
+            MoveOnUpdate();
+        }
+
+        protected abstract void MoveOnUpdate();
+    }
+}
```

Comment:
Please add new line to the end of the file.

Reply:
Weird, it's there; GitHub just isn't showing it for some reason.